### PR TITLE
Move CI tests to GitHub actions

### PR DESCRIPTION
**Pull request by jbygdell**
### Describe the pull request:

- [ ] Bug fix
- [X] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

Moves our CI tests from travis to Github Actions

### Changes made:

1) Linter for both charts
2) E2e-test using k3s
3) Small updates to both charts to be compliant with k8s 1.15+ api changes.
### Additional information:

Kubernets api has changed from 1.15+ so some changes here reflect that.
Rework on the charts will be needed to ensure that all options are working as they should.

### CODE REVIEW

File: .github/workflows/k3s.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      matrix:
+        python-version: [3.6]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Clone and install deploy-inint
```
Comment (blankdots) [inline .github/workflows/k3s.yml]:
```suggestion
    - name: Clone and install deploy-init
```

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      matrix:
+        python-version: [3.6]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Clone and install deploy-inint
```
Comment (blankdots) [inline .github/workflows/lint.yml]:
```suggestion
    - name: Clone and install deploy-init
```

File: .travis.yml [hunk 3]
```diff
@@ -11,113 +11,11 @@ env:
     - HELM_URL=https://storage.googleapis.com/kubernetes-helm/helm-v2.14.1-linux-amd64.tar.gz
```
Comment (blankdots) [inline .travis.yml:11]: should we switch here to v3 as well?

**Review by blankdots — CHANGES_REQUESTED** [review]
small changes

File: .travis.yml (hunk 3 above)
Comment (jbygdell) [reply .travis.yml:11]: Not yet, I really don't see the need for building the packaged chart when you just can clone the repo.

**Review by blankdots — APPROVED** [review]

**Pull request merged**